ci-operator/step-registry/hypershift/hostedcluster/create/hostedcluster/hypershift-hostedcluster-create-hostedcluster-commands.sh

```diff
@@ -37,6 +37,12 @@ else
   exit 1
 fi
 
+# TODO: BASE_DOMAIN is hardcoded to aws account, we need to change it
+# based on which account this job is running.
+if [[ "${CLUSTER_PROFILE_NAME}" != 'aws' ]]; then
```

Comment:
`[[ ! -z "$BASE_DOMAIN" ]]`

This is missing if a job sets the BASE_DOMAIN then it should not be replaced